src/playground/src/ui/screen/tabNavigator/tabNavigator.component.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import {
+  StyleSheet,
+  View,
+  ViewProps,
+} from 'react-native';
+import {
+  createBottomTabNavigator,
+  NavigationContainer,
+  NavigationContainerProps,
+  NavigationScreenProp,
+  NavigationState,
+  NavigationRoute,
+} from 'react-navigation';
+import {
+  BottomTabNavigator as BottomTabNavigatorComponent,
+  TabRoute,
+} from '@kitten/ui';
+import { TabScreen1 } from './tabScreen1.component';
+import { TabScreen2 } from './tabScreen2.component';
+import { TabScreen3 } from './tabScreen3.component';
+
+interface NavigationProps {
+  navigation: NavigationScreenProp<NavigationState>;
+}
+
+type CommonNavigationProps = NavigationProps & NavigationContainerProps;
+
+export const TabNavigatorScreen: NavigationContainer = createBottomTabNavigator({
+  Screen1: TabScreen1,
+  Screen2: TabScreen2,
+  Screen3: TabScreen3,
+}, {
+  initialRouteName: 'Screen1',
+  tabBarComponent: (props: CommonNavigationProps) => getTabNavigationComponentWithRoutes(props),
+});
+
+function getTabNavigationComponentWithRoutes(props: CommonNavigationProps): React.ReactElement<ViewProps> {
```

Comment:
This method looks like a regular `render`, consider refactoring it to different name

Reply:
Will be refactored.